application-services/custom/camera-management/README.md

```diff
@@ -107,13 +108,25 @@ sudo apt install build-essential
 
    b. Under the `ports` section, find the entry for port 8554 and change the host_ip from `127.0.0.1` to either `0.0.0.0` or the ip address you put in the previous step.
 
-6. Run the following `make` command to run the edgex core services along with the Onvif and Usb device services. 
+6. Run the following `make` command to generate the edgex core services along with MQTT, Onvif and Usb device services.
 
   > **Note**: The `ds-onvif-camera` parameter can be omitted if no Onvif cameras are present, or the `ds-usb-camera` parameter can be omitted if no usb cameras are present.
 ```shell
-   make run no-secty ds-onvif-camera ds-usb-camera 
+   make gen no-secty ds-mqtt mqtt-broker ds-onvif-camera ds-usb-camera 
 ```   
 
+7. Configure [device-mqtt] service to send [Edge Video Analytics Microservice][evam] inference results into Edgex via MQTT
+
+      a. Copy complete [evam-mqtt-edgex](edge-video-analytics/evam-mqtt-edgex) folder into edgex-compose/compose-builder directory.
+
+      b. Copy and paste [docker-compose.override.yml](edge-video-analytics/evam-mqtt-edgex/docker-compose.override.yml) from the above copied folder into edgex-compose/compose-builder directory.
+         Insert full path of `edgex-compose/compose-builder` directory under volumes in this `docker-compose.override.yml`.
+
+
+8. Run the following command to start all the Edgex services.
+```shell
+   docker compose -f docker-compose.yml -f docker-compose.override.yml up -d
```

Comment:
Should we add `-p edgex` to be safe, or do we know even with Levski branch it works out of the box with that name?

Reply:
Yes. It works will all release since compose builder was added.